backend/learnings/tests.py

```diff
@@ -1,16 +1,17 @@
 from django.contrib.auth.models import User
+from django.urls import reverse
 from rest_framework.authtoken.models import Token
 from rest_framework.test import APITestCase
 
 from .models import Category
 
 
 class CategoryAPIViewTests(APITestCase):
-    add_category_url = "http://127.0.0.1:8000/add_category/"
-    category_list_url = "http://127.0.0.1:8000/category_list/"
-    category_per_user_url = "http://127.0.0.1:8000/category/"
-    delete_category_url = "http://127.0.0.1:8000/delete_category/"
-    edit_category_url = "http://127.0.0.1:8000/edit_category/"
+    add_category_url = reverse("add_category")
+    category_list_url = reverse("category_list")
+    category_per_user_url = reverse("category_per_user", args=["Becs"])
+    delete_category_url = reverse("delete_category", args=[1])
+    edit_category_url = reverse("edit_category", args=[1])
```

Comment:
sorry i forgot to mention earlier, please move this inside the setup :bow: 
because there are times when we need some data from the setup to be passed as args/kwargs